ballista/core/proto/datafusion.proto

```diff
@@ -21,7 +21,7 @@
  package datafusion;
  
  option java_multiple_files = true;
- option java_package = "org.apache.arrow.datafusion.protobuf";
```

Comment:
this file is coped from datafusion, I believe we should change it there 

https://github.com/apache/datafusion/blob/7ef5c4b291bdb13d5b53e7e344c7a6ec1cb2c8b4/datafusion/proto/proto/datafusion.proto#L24

Reply:
Thanks. I also created https://github.com/apache/datafusion/pull/15658